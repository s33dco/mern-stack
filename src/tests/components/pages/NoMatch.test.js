import React from 'react';
import { shallow } from 'enzyme';
import NoMatch from '../../../components/pages/NoMatch';

let wrapper;

beforeEach(() => {
	wrapper = shallow(<NoMatch />);
});

describe('Home component', () => {
	it('should render NoMatch component correctly', () => {
		expect(wrapper).toMatchSnapshot();
	});
});
