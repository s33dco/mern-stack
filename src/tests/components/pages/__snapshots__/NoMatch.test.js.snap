// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMatch component should render NoMatch component correctly 1`] = `ShallowWrapper {}`;
